sway-error/src/parser_error.rs

```diff
@@ -94,6 +94,8 @@ pub enum ParseErrorKind {
     UnexpectedTokenAfterSliceType,
     #[error("Expected a path type.")]
     ExpectedPathType,
+    #[error("Expected ':'. Enum variants must be on the form 'Foo : ()' or 'Bar : u32'.")]
```

Comment:
```suggestion
    #[error("Expected ':'. Enum variants must be in the form `Variant: ()`, `Variant: <type>`, or `Variant: (type1, ..., typeN)`")]
```

Reply:
Let's do it like Rust does it, by listing all the possible forms and using the generic name "Variant". Note that according to the diagnostics RFC we enclose code in ``.